releng/tools/org.openjdk.jmc.util.releasenotes/notes/8.2/notes.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<?xml-stylesheet type="text/xsl" href="stylesheet.xsl"?>
+<notes>
+	<component>
+		<!-- ******************************* GENERAL ****************************************** -->
+		<name>General</name>
+		<note version="8.2">
+			<title>JMC 8.2 - New Release!</title>
+			<description>
+				This is a new minor release of Java Mission Control. The
```

Comment:
Should this be "JDK Mission Control" instead of "Java Mission Control"? At least on the GitHub open source notes.